posts/2020/02/my-unexpected-dive-into-open-source-python.md

```diff
@@ -0,0 +1,61 @@
+
+![PyData NYC Banner](/images/codesprite-comics/PyData-NYC-Banner.png)
+
+When I first heard about [PyData](https://pydata.org/) and it's [Diversity Scholar](https://pydata.org/diversity-inclusion/) program, I didn't think either applied to me. I didn't feel like I knew enough Python nor deserve a place to learn more about it. I'm always eager to learn something new, but PyData felt too out of my reach.
+
+Thankfully, my friend who was a volunteer at PyData really encouraged me to apply. I gave it a shot. I was rejected at first - I didn't quite make the cut. I was disappointed and accepted it.
+
+But a second round of funding came in from partner organizations. Now, they could bring in more Diversity Scholars. I was elated to have the chance! Their trust in me gave me the confidence to go. It totally changed my point of view from self-doubt to eagerness.
+
+Once I was at PyData, I signed up for the Lightning Talks, knowing that it would be a great way for me to talk about my personal project, the 'Codesprite Explains' comics.
+
+The 'Codesprite' comics break down code concepts into fun and visual ways. It was inspired by my desire to turn my boring written notes into something more personal.
+
+![Page 1 of first Codepsrite comic](/images/codesprite-comics/CodeSprite-part-1.png) ![Page 2 of first Codepsrite comic](/images/codesprite-comics/CodeSprite-part-2.png)
+*First two pages of my personal project, 'Codesprite Explains'*
+
+At my Lightning Talk, I was confident. **No longer did I doubt that I could be here- to take up space and stand up tall.** I introduced myself as a Diversity Scholar. I was unabashedly unashamed to be new to Python and eager to learn. I delighted in showing off my artistic skills that give me so much joy and that aren’t common in this community. I laughed with the crowd at my cheesy puns.
+
+When I finished my talk, I realized I came a long way from where I started. So when Ralf congratulated me, I was already feeling good. When he asked if I was interested in joining Quansight, that was the most unexpected-yet-awesomest cherry on top!
+
+![Author giving a Lightning Talk at PyData NYC](/images/codesprite-comics/PyData-Lightning-Talk.png)
+*The author giving a Lightning Talk at PyData NYC*
+
+In my next blog post, I'll talk about my first impressions of Quansight and what made me say 'yes'.
+
+Before ending this post, I would like to recap a few things.
+
+**There were some initial barriers that prevented me from joining the open-source space.** Without knowing people involved in the community, such as my friend volunteering there, I wouldn't have known about PyData. Without the financial assistance of the Diversity Scholar program, I wouldn't have been able to afford a ticket as an unemployed graduate. A lack of technical knowledge itself isn't prohibitive, but when combined with the above factors, stops many other capable people from entering this space.
```

Comment:
minor textual: "other capable" --> "otherwise capable" ?

Reply:
I think the difference is
"other capable"- that there's more people out there who could join
"otherwise capable"- when these barriers are put down, these capable people could join

Now I'm not sure myself which one to use now since the difference is minor...

Reply:
I decided to stick with 'other'